### ISSUE TITLE
Render mob in factory

### ISSUE THREAD

**Issue by Ipsis** [Ipsis/Woot#20]
Can we render the programmed mob somewhere within the factory structure.
eg. TileEntityMobSpawnerRenderer/renderMob

**Comment by ShadowDrakken** [comment]
This would be neat. I could just picture the main mob containing block looking a bit like an encased vanilla mod spawner like this :)

**Comment by Protheus** [comment]
Yeah this is great, as then you can see in one glance which mob is currently being killed.

**Comment by kyle-mathis** [comment]
Maybe have it able to be set to on or off in the config to aid in those who want to keep lag down on a server

**Comment by Ipsis** [comment]
Chances of my adding this are pretty low. I think it was a good idea at the time, but to be honest, I had writing rendering code. So this is way down low on the priority list, but if ever implemented would definitely have a config option.
